src/Struct/FindologicProduct.php

```diff
@@ -356,11 +356,17 @@ public function getKeywords(): array
 
     protected function setKeywords(): void
     {
-        $tags = $this->product->getTags();
-        if ($tags !== null && $tags->count() > 0) {
-            foreach ($tags as $tag) {
-                if (!Utils::isEmpty($tag->getName())) {
-                    $this->keywords[] = new Keyword($tag->getName());
+        $ignoreKeywords = [$this->product->getProductNumber(), $this->product->get('id'), $this->product
+        ->getManufacturer()->getName()];
```

Comment:
- Use `$manufacturer->getTranslation('name')` instead of getting name directly, see 
`\FINDOLOGIC\FinSearch\Struct\FindologicProduct::setVendors` method for reference. 
- No need to use `$product->get('id')`, you can use `getId()` directly, provided that we need that in the ignored list in the first place. See other comment about it.